README.md

```diff
@@ -1,31 +1,70 @@
-# This extension is barely maintained anymore, new maintainers are welcome!
+# gnome-shell-extension-caffeine
 
-## gnome-shell-extension-caffeine
-  - Fill the cup to inhibit auto suspend and screensaver
-  - This extension supports GNOME Shell `3.4` -> `43`
-    - `master`: `40` -> `43`
-    - `gnome-shell-3.36-3.38`: `3.36` -> `3.38`
-    - `gnome-shell-3.32-3.34`: `3.32` -> `3.34`
-    - `gnome-shell-3.10-3.30`: `3.10` -> `3.30`
-    - `gnome-shell-before-3.10`: `3.4` -> `3.8`
+[<img src="https://github.com/pakaoraki/gnome-shell-extension-caffeine/raw/master/ressources/get_it_on_gnome_extensions.png" height="100" align="right">](https://extensions.gnome.org/extension/517/caffeine/)
```

Comment:
Should this link be to "https://github.com/eonpatapon/..." ?

Reply:
Thanks for your feedback.
folder name fixed.

"Should this link be to "https://github.com/eonpatapon/..." ?"
I did it on purpose as the resource does not exist yet on the main repo. That's a small change to do after merging.